### PR TITLE
feat(theme): final re-brand

### LINKED ISSUE #2610: No contrast of code on table row in light theme
### Describe the bug

In the light theme, a span of formatted code has the same background color as the striped rows of the table.

| Element | Var | Value in light theme |
| --- | --- | --- |
| .vp-doc tr:nth-child(2n) | `--vp-c-bg-soft` | #f6f6f7 |
| :not(pre) > code | `--vp-c-mute` | #f6f6f7 |

### Reproduction

Render this table with the default theme in light in VitePress:

```md
| Element | Var | Value in light theme |
| --- | --- | --- |
| .vp-doc tr:nth-child(2n) | `--vp-c-bg-soft` | #f6f6f7 |
| :not(pre) > code | `--vp-c-mute` | #f6f6f7 |
```

### Expected behavior

There is a contrast between the background of the code span and all table rows, like here on GitHub.

### System Info

```sh
System:
    OS: Windows 10 10.0.19044
    CPU: (16) x64 Intel(R) Xeon(R) W-11955M CPU @ 2.60GHz
    Memory: 13.44 GB / 31.73 GB
  Binaries:
    Node: 18.12.0 - C:\Program Files\nodejs\node.EXE
    npm: 9.8.0 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: Spartan (44.19041.1266.0), Chromium (114.0.1823.79)
  npmPackages:
    vitepress: ^1.0.0-beta.5 => 1.0.0-beta.5
```

### Additional context

Suggestion: tone down the mute colors in the light theme a bit. This makes code spans also more visible on the default white background.

```scss
:root {
  --vp-c-mute: #eeeef0; // luv = 94
  --vp-c-mute-light: #f3f3f9; // luv = 96
  --vp-c-mute-lighter: #f9f9fa; // luv = 98
}
```

[HSLuv system](https://www.hsluv.org/) used.

### Validations

- [X] Check if you're on the [latest VitePress version](https://github.com/vuejs/vitepress/releases/latest).
- [X] Follow our [Code of Conduct](https://vuejs.org/about/coc.html)
- [X] Read the [docs](https://vitepress.dev).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.

### PR DESCRIPTION
**Pull request by kiaking**
- close #2610 

The final re-branding before going RC. The VitePress brand is now aligns with Vite.

I have tweak mostly the CSS vars but tiny other parts that I wonted adjust a bit. Now we have 2 separate colors for Light/Dark mode. We should be eliminate a11y problems we have or would have in the future.

- [x] Add social preview images

---

Screenshot 2023-08-03 at 21 13 51

Screenshot 2023-08-03 at 21 14 05

Screenshot 2023-08-03 at 21 16 00

Screenshot 2023-08-03 at 21 15 40

### CODE REVIEW

**Review by brc-dd — APPROVED** [review]
updated some stuff to improve perf 🚀

File: src/client/theme-default/styles/vars.css [hunk 1]
```diff
@@ -86,31 +58,67 @@
...
 
-  --vp-c-brand: var(--vp-c-green);
-  --vp-c-brand-light: var(--vp-c-green-light);
-  --vp-c-brand-lighter: var(--vp-c-green-lighter);
-  --vp-c-brand-dark: var(--vp-c-green-dark);
-  --vp-c-brand-darker: var(--vp-c-green-darker);
-
   --vp-c-mute: #f6f6f7;
   --vp-c-mute-light: #f9f9fc;
   --vp-c-mute-lighter: #ffffff;
```
[the remark this answers is not in the document]
Comment (kiaking) [reply src/client/theme-default/styles/vars.css]: Updated! 👍

**Comment by kiaking** [comment]
Thanks for the tweak! I don't know why but I converted png to webp and it got even smaller (33kb) so I've replaced it with that 🤔  And fixed some border color issue on CSS var.

**Comment by zonemeen** [comment]
Great great great job! Look so cool !👀

**Comment by brc-dd** [comment]
> I don't know why but I converted png to webp and it got even smaller (33kb)

Yeah, I too expected it to be smaller. But pixelmator generated that 50 kB or something.

**Comment by Lehoczky** [comment]
Shall we adjust the text color for the danger containers `(--vp-custom-block-danger-text)` to meet the AA contrast ratio? 

I found the following suggestions using [chrome devtools](https://developer.chrome.com/docs/devtools/accessibility/contrast/#fix-low-contrast), that are very close to the current color:
- for **dark** theme: `#ff4666` instead of `#f43f5e` 
- for **light** theme: `#cd354f` instead of `#f43f5e`

Right now we use the same color for both light and dark themes, but unfortunately there is no shade of red which meets the AA contrast requirement for both themes:

image
image

First picture is for dark, second is for light theme.

**Comment by brc-dd** [comment]
@kiaking close #2119 as well if the container colors are adjusted 🙌

PS: we can also pick colors from https://www.radix-ui.com/colors, their grouping seem to yield 5-10 contrast ratio (use 9/10/11 for text, 1/2 for bg)

File: src/client/theme-default/styles/components/vp-doc.css [hunk 2]
```diff
@@ -109,13 +109,10 @@
...
   font-weight: 500;
   color: var(--vp-c-brand);
-  text-decoration-style: dotted;
-  transition: color 0.25s;
 }
 
 .vp-doc a:hover {
-  /* color: var(--vp-c-brand-dark); */
-  text-decoration: underline;
+  text-decoration: underline dotted;
```
Comment (Lehoczky) [inline src/client/theme-default/styles/components/vp-doc.css:115]: Just a suggestion, I think moving the dotted line down a bit looks a bit better. What's do your opinion?

```suggestion
  text-decoration: underline dotted;
  text-underline-offset: 2px;
```

image
image

**Comment by userquin** [comment]
@brc_dd do we have some codeblock inside `::: <tip|info|warning|error>` in the playground? We should check color constrast, something like this https://vite-pwa-org.netlify.app/guide/auto-update.html#automatic-reload.

**Comment by brc-dd** [comment]
> do we have some codeblock inside `::: <tip|info|warning|error>` in the playground?

By playground, do you mean e2e tests? There aren't a11y checks there right now. We can add those, but currently they will fail. They shouldn't be necessary IMO, we won't later be changing the colors much.

**Comment by userquin** [comment]
I mean, with a simple visual test va dev `tools.,` there is no need to add `a11y` tests

**Comment by brc-dd** [comment]
What's playground? The deploy preview https://deploy-preview-2727--vitepress-docs.netlify.app/guide/markdown#custom-containers ?

**Comment by kiaking** [comment]
Added social preview image 👍

**Pull request merged**